utils/rpms/daos.spec

```diff
@@ -3,7 +3,7 @@
 %define agent_svc_name daos_agent.service
 %define sysctl_script_name 10-daos_server.conf
 
-%global mercury_version 2.3.1-1%{?dist}
+%global mercury_version   2.4
```

Comment:
it does not need to be explicit, I just want whatever is above 2.4

Reply:
> it does not need to be explicit, I just want whatever is above 2.4

Excellent!  This is what I like to see.  But just to make doubly sure you are getting what you want: https://github.com/daos-stack/daos/blob/4320e9cab22518d1a1ddd2d6308daf0d218b69d3/utils/rpms/daos.spec#L31
means you will be getting _whatever is equal to or above 2.4_.  I suspect that is what you want, but wanted to point it out just in case.

Reply:
yes this is what I want :)